packages/core/src/components/forms/_label.scss

```diff
@@ -8,19 +8,22 @@
 /*
 Labels
 
-Add labels to inputs to enhance the usability of your forms.
+Labels enhance the usability of your forms.
 
-Putting the `<input>` element _inside_ a `<label>` element also increases the area where the user
-can click to activate the control. Notice how in the example below, clicking a `<label>` below
-focuses its `<input>`.
+- Add extra information to the label with `span.pt-label-description`.
+
+- Putting the `<input>` element _inside_ a `<label>` element increases the area where the user
+can click to activate the control. Notice how in the examples below, clicking a `<label>` focuses its `<input>`.
```

Comment:
We actually need to make a decision whether or not we want to deprecate the older way of writing forms (`input` inside `label`). Once we decide, we can put the appropriate wording in the docs 👍 

@giladgray @adidahiya @cmslewis See above, need your thoughts. Do we:
1. Deprecate the older way of writing forms (`input` inside `label`)
1. Keep the older way (`input` inside `label`) alongside the new way (`pt-form-group` + `for` + `id`) and maintain these two approaches over time.

I see a valid use case for keeping both around: `input` inside `label` can be used for dead simple form fields, where `pt-form-group` should be used for more complex ones (i.e. ones that require control groups for instance.)

I can also see a valid reason to remove the super simple way: it doesn't support all the use cases (no way to make control group work cleanly inside a `label`), and it's easier to maintain one approach.